modules/alias/outputs.tf

```
@@ -23,3 +23,23 @@ output "lambda_alias_function_version" {
   description = "Lambda function version which the alias uses"
   value       = try(data.aws_lambda_alias.existing[0].function_version, aws_lambda_alias.with_refresh[0].function_version, aws_lambda_alias.no_refresh[0].function_version, "")
 }
+
+output "lambda_alias_event_source_mapping_function_arn" {
+  description = "The the ARN of the Lambda function the event source mapping is sending events to"
+  value       = length(aws_lambda_event_source_mapping.with_refresh) > 0 ? { for k, v in aws_lambda_event_source_mapping.with_refresh : k => v.function_arn } : { for k, v in aws_lambda_event_source_mapping.no_refresh : k => v.function_arn }
```

```suggestion
  value       = { for k, v in aws_lambda_event_source_mapping.this : k => v.function_arn }
```

and similar to other outputs